tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerTests.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.Bot.Builder.Dialogs.Adaptive;
```

Comment:
These are all Adaptive tests. They is not an adaptive assembly. The QnaMakerDialog class isn't (that I can tell) in the Adaptive namespace. 

This layering probably isn't quite right.